src/prefect/cli/root.py

```diff
@@ -74,9 +77,14 @@ def main(
             exit(1)
 
     # Configure the output console after loading the profile
-
-    app.console.is_interactive = prompt
-    app.console.soft_wrap = not PREFECT_CLI_WRAP_LINES.value()
+    app.console = rich.console.Console(
```

Comment:
Oof, good catch.  The big reason I moved it over to the `PrefectTyper` was because there was no declared attribute `PrefectTyper.console`, so the typing of almost every CLI module was busted.  Thanks for the fix!

One thought: what if we move this to a function so it's like:

`app.setup_console()`

and then over in `PrefectTyper`:

```python
class PrefectTyper:
    console: rich.console.Console

    ...

    def setup_console(self, prompt: bool):
        self.console = rich.console.Console(...)
```

This just might keep things a little more together